### ISSUE TITLE
[Question/Feature Request] Dashboard Customisation

### ISSUE THREAD

**Issue by vaz-lenbox** [farridav/django-jazzmin#335]
Can we override the project / app admin index and customize it? How can I extend django-jazzadmin to add custom features like charts or define custom ModelAdmins or forms for a custom interface?

If it is possible, can you add examples in the documentation ?

**Comment by youssef02** [comment]
Yeah I was thinking the same, the dashboard is static, from what I found on base.html, someone need to implement it because I think it will be a good core feature, but again this issue is open for some time i guess
